src/components/ResultsCount.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <span v-if="isNumber(count)" class="results-count">
-    {{ count }} {{ toPluralString(label, count) }}
+    {{ count.toLocaleString() }} {{ toPluralString(label, count) }}
```

Comment:
This fixes a widespread issue where results counts > 999 wouldn't display nicely

Reply:
nice, good catch